MANIFEST.in

```diff
@@ -0,0 +1,11 @@
+include requirements/*.txt
+include mmtrack/VERSION
+include mmtrack/model_zoo.yml
+include mmtrack/configs/*/*.py
+include mmtrack/configs/*/*.yml
+include mmtrack/configs/*/*/*.py
+include mmtrack/configs/*/*/*.yml
+include mmtrack/tools/*.py
+include mmtrack/tools/*.sh
+include mmtrack/tools/*/*.py
+include mmtrack/demo/*
```

Comment:
1. Use recursive-include to update the code, refer [here](https://github.com/open-mmlab/mmdetection/commit/69d4b4d485934c4264a0019f429ab26fb52bb118)
2. Add an empty line at the end of the file.

Reply:
I have fixed these two issues.